sqlparse/filters.py

```diff
@@ -690,9 +690,13 @@ def _create_column_definition(self, token_list):
         column_definition_children_tokens.extend(
             self._skip_white_space_and_new_line_tokens(token_queue)
         )
-        column_type_length_token = self._create_column_type_length(token_queue)
-        if column_type_length_token:
-            column_definition_children_tokens.append(column_type_length_token)
+
+        if col_type_token.value == 'set' or col_type_token.value == 'enum':
+            column_type_meta_token = self._create_column_type_values(token_queue)
```

Comment:
nitpick: `meta_token` may not be a good name for these tokens, not sure what it means in the sql world.